examples/README.md

```diff
@@ -8,85 +8,159 @@ title: Examples
+### Interactive shell script
 
-```sh {"id":"01HF7B0KK32HBQ9X4AAT0019KB"}
-node ./scripts/stdin.js
-```
+You can write interactive (Stdin) programs too without leaving your Markdown file!
+Since this is a long running process, ensure we've marked the cell as a background process, otherwise the cell execution will block the Notebook UX, you can configure that at the cell level configuration section, check the background option from the advanced tab. [Learn more here](https://docs.runme.dev/configuration/cell-level#handle-long-running-processes).
 
-## Mix & Match Languages
+Feel free to run the following shell script example, it's similar to the above example with the main difference is now interactive!, it will give you the following information about your system:
+ - Disk usage
+ - Memory usage
+ - CPU load
 
-You can also execute JavaScript inline:
+```sh {"background":"true","name":"interactive-shell-script","terminalRows":"25"}
+#!/bin/sh
 
-```js {"id":"01HF7B0KK32HBQ9X4AAW385HPB"}
-(function({ message }) {
-    console.log(message)
-})({ message: 'Running javascript that outputs this message' })
-```
+check_disk_usage() {
+    echo "Disk Usage:"
+    df -h
+}
+
+check_memory_usage() {
+    echo "Memory Usage:"
+    top -l 1 -s 0 | grep PhysMem | sed 's/, /\n         /g'
+}
+
+check_cpu_load() {
+    echo "CPU Load:"
+    uptime
+}
 
-Or typescript:
+display_menu() {
+  echo -e "_________________________________________________________________\n"
+  echo "Hello! What operation you want to perform? (press ctrl+c to exit)"
+  echo "1. Memory Usage"
+  echo "2. Disk Usage"
+  echo "3. CPU Load"
+  echo -e "_________________________________________________________________\n"
+  echo "Type the option number and hit enter"
+}
 
-```typescript {"id":"01HF7B0KK32HBQ9X4AAXEVS9QR"}
-function unnest({ message }: { message: string }): void {
-    console.log(message)
+# Function to handle Ctrl+C
+ctrl_c_handler() {
+    echo "Exiting..."
+    exit 0
 }
 
-unnest({ message: 'Running typescript that outputs this message' })
+# Trap Ctrl+C signal
+trap ctrl_c_handler SIGINT
+
+while true; do
+  display_menu
+  read option
+
+  case $option in
+      1)
+          echo " >> You selected: Memory Usage"
+          check_memory_usage
+          ;;
+      2)
+          echo " >> You selected: Disk Usage"
+          check_disk_usage
+          ;;
+      3)
+          echo " >> You selected: CPU Load"
+          check_cpu_load
+          ;;
+      *)
+          echo "Invalid option selected!"
+          ;;
+  esac
+done
 ```
 
-Please see more examples, including configuration languages further down.
+## Complex Output
+
+The integrated Runme terminal is able to handle complex outputs too.
+
+Take a look at the following example:
+
+### Install kubectl
+
+```sh {"background":"true","name":"install-kubectl","terminalRows":"25"}
+curl -# -v -o /dev/null "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/darwin/arm64/kubectl"
```

Comment:
what's going on here? this isn't running for me

Reply:
Works now, @degrammer. However, I am wondering if we should just assume `kubectl` is already installed and do a simple helm flow? https://helm.sh/docs/intro/quickstart/?

Pulling in a chart ("deployment plan") and installing it in a cluster?

Reply:
@degrammer btw, Eddie's been drafting a guide for Helm which you could use as inspiration: https://docs-runme-dev-git-t-helm-stateful.vercel.app/guide/helm